test/configCases/container/reference-hoisting/index.js

```diff
@@ -0,0 +1,15 @@
+it("should load the component from container", () => {
+	return import("./App").then(({ default: App }) => {
+		const rendered = App();
+		expect(rendered).toBe(
+			"App rendered with [This is react 0.1.2] and [ComponentA rendered with [This is react 0.1.2]]"
+		);
+		return import("./upgrade-react").then(({ default: upgrade }) => {
+			upgrade();
+			const rendered = App();
+			expect(rendered).toBe(
+				"App rendered with [This is react 1.2.3] and [ComponentA rendered with [This is react 1.2.3]]"
+			);
+		});
+	});
+});
```

Comment:
Can we do more better testing here because this is just a test for the working rather than the actual behaviour?

Reply:
yeah i updated the test, i was blocked on the original issue being unable to import module federation plugin for the hooks